MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs

```diff
@@ -662,13 +661,13 @@ private ChapterInfo[] CreateDummyChapters(Video video)
                         runtime));
             }
 
-            if (runtime < _dummyChapterDuration)
+            if (runtime < dummyChapterDuration)
             {
                 return Array.Empty<ChapterInfo>();
             }
 
             // Limit to 100 chapters just in case there's some incorrect metadata here
```

Comment:
update comment

Reply:
Hello, could you check my last commit? I modified the comment.